hivemind/p2p/p2p_daemon.py

```diff
@@ -68,10 +66,12 @@ def __init__(self):
     @classmethod
     async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager: bool = True,
                      dht_mode: str = 'dht_server', force_reachability: Optional[str] = None,
-                     nat_port_map: bool = True, auto_nat: bool = True, bootstrap: bool = False,
-                     bootstrap_peers: Optional[List[str]] = None, use_global_ipfs: bool = False, host_port: int = None,
```

Comment:
`host_port` parameter is misleading:

1. A user may expect that it contains both a host and a port, while it actually contains only a port.
2. It is unclear which port it defines (external p2p port, local daemon port, or local daemon controller port).

I hope the name `p2p_port` solves these problems.